src/defaultsettings.cpp

```diff
@@ -57,6 +57,10 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("curl_verify_cert", "true");
 	settings->setDefault("enable_remote_media_server", "true");
 	settings->setDefault("enable_client_modding", "false");
+	settings->setDefault("enable_knockback", "true");
+	settings->setDefault("knockback_default_duration", "0.3");
```

Comment:
But a timer is important for the server. The sending of the player's position to the other players  is blocked during knockback because each client makes the animation.